packages/embark/src/lib/modules/blockchain_process/gethClient.js

```diff
@@ -135,6 +139,11 @@ class GethClient {
     return cmd;
   }
 
+  isTestNetwork(config) {
+    return ['testnet', 'rinkeby'].includes(config.networkType) ||
```

Comment:
Is it possible to extract network names and network IDs out to the constants file?

Reply:
👍

Reply:
Totally, good call.